examples/protocols/displayport/displayport-board.stanza

```diff
@@ -0,0 +1,202 @@
+#use-added-syntax(jitx)
+defpackage jsl/examples/protocols/displayport/displayport-board :
+  import core
+  import jitx
+  import jitx/commands
+
+  import jsl/protocols/displayport
+
+
+public pcb-rules displayport-board-rules :
```

Comment:
Is there a way we can reuse the board setup from the PCIe example so that we don't have this copy-pasta ? 

I create the PCIe board because the board that I had in `examples/landpatterns/board.stanza` was not sufficient for the PCIe signal integrity setup. 

But it seems like PCIe and Display-Port could share the stackup at least and then maybe have slightly different routing structures ?

Reply:
I suppose that we can do that -- my goal is to have a standalone example when users want to adapt the code to use it in their designs. If we move the board code out of each example, then they will have to root around to find it to copy it.

Reply:
And I suspect that for the first examples that we are going to have a lot of code-sharing, but I can see later examples might diverge especially if we get via-structures.

Reply:
I think that kind of duplication is fine `jitx-cookbook`

I don't want to have to maintain duplication in JSL.

Reply:
Done.